src/lib/marshal/translation/processor.isomorphic.ts

```diff
@@ -1,3 +1,4 @@
+import { DEFAULT_TRANSLATION_FORMAT, TranslationFormat } from "./helpers";
```

Comment:
I'm pretty sure this will break the package when used in Control because we are now importing from ./helpers which uses `fs` and other modules that are node specific. Admittedly this is not obvious at all, I thought I had a eslint rule but I guess that was for a different check.

I think the move is to move `TranslationFormat` to `./types`, and `DEFAULT_TRANSLATION_FORMAT` to this module and import from here.

Reply:
Good catch, I didn't realize we were using the cli in the dashboard. Good to know! Yeah, I'll move those to the isomorphic module.

Reply:
Yep we use it for showing commit diffs in dashboard, because the diffs that get rendered are based on the file representation of workflow.json, layout.json etc. And all of that (non trivial) logic is contained in the CLI and I didn't want to duplicate that. Which I think is kinda cool, using our own package! :)